portfolio/src/main/webapp/WEB-INF/bird-species-by-country.csv

```diff
@@ -0,0 +1,231 @@
+French Guiana,709
+Guatemala,709
+Honduras,704
+Suriname,700
+Laos,698
+Paraguay,691
+Ghana,681
+Nicaragua,677
+Mozambique,675
+Ivory Coast,670
+Russia,661
+Rwanda,638
+Malawi,636
+Guinea,633
+Zimbabwe,627
+Bhutan,615
+Pakistan,611
+CG,611
+Bangladesh,607
+Gabon,606
+Namibia,599
+Burundi,596
+Philippines,593
+Sierra Leone,582
+Togo,573
+Somalia,570
+Mali,566
+Senegal,556
+Belize,549
+Benin,548
+Eritrea,540
+Liberia,537
+Botswana,534
+Chad,524
+Canada,518
+Cambodia,512
+El Salvador,500
+Iran,474
+Mauritania,471
+Swaziland,463
+Guinea-Bissau,456
+Gambia,455
+Burkina Faso,450
+Japan,446
+Chile,441
+Kazakhstan,440
+Equatorial Guinea,436
+Niger,436
+Trinidad and Tobago,414
+British Indian Ocean Territory,411
+Uruguay,408
+Turkey,392
+Afghanistan,391
+Saudi Arabia,390
+Israel,388
+Spain,382
+Egypt,378
+Taiwan,377
+Sri Lanka,376
+Iraq,374
+Singapore,366
+Turkmenistan,365
+Mongolia,360
+South Korea,359
+France,357
+Italy,355
+Uzbekistan,352
+Azerbaijan,349
+Greece,346
+Yemen,342
+Morocco,341
+Bulgaria,335
+Jordan,331
+Kyrgyzstan,328
+Tajikistan,326
+North Korea,325
+Oman,323
+Romania,322
+Ukraine,317
+Algeria,317
+Cuba,317
+Serbia,312
+Germany,311
+Montenegro,310
+Portugal,310
+Tunisia,309
+Croatia,308
+Austria,304
+Albania,302
+Armenia,296
+Lebanon,295
+Slovakia,293
+United Arab Emirates,293
+Kuwait,292
+Poland,290
+Slovenia,289
+Djibouti,287
+Czech Republic,286
+Hungary,285
+Switzerland,284
+Bosnia and Herzegovina,281
+Georgia,281
+Sweden,277
+Denmark,275
+Cyprus,275
+Netherlands,270
+Puerto Rico,268
+Latvia,267
+United Kingdom,266
+Libya,266
+Bahamas,264
+Finland,263
+Belarus,263
+Belgium,263
+Haiti,258
+Norway,254
+Estonia,253
+Lesotho,248
+Madagascar,247
+Dominican Republic,246
+Solomon Islands,245
+Turks and Caicos Islands,242
+Lithuania,242
+East Timor,236
+Qatar,233
+New Zealand,231
+Bahrain,222
+Hong Kong,218
+Martinique,217
+Saint Lucia,208
+Ireland,208
+Barbados,207
+Antigua and Barbuda,206
+Dominica,206
+Malta,205
+Saint Vincent and the Grenadines,201
+Saint Pierre and Miquelon,199
+Saint Kitts and Nevis,199
+Montserrat,197
+Luxembourg,186
+Jamaica,186
+Guadeloupe,183
+Aruba,182
+Liechtenstein,176
+Western Sahara,165
+Gibraltar,163
+Bermuda,152
+U.S. Virgin Islands,148
+British Virgin Islands,147
+New Caledonia,126
+Andorra,125
+Faroe Islands,121
+Anguilla,120
+Northern Mariana Islands,116
+Palau,113
+Grenada,111
+Iceland,109
+Fiji,109
+Guam,106
+Comoros,100
+Seychelles,98
+French Polynesia,94
+United States Minor Outlying Islands,87
+Cape Verde,87
+Vanuatu,87
+Maldives,84
+Mauritius,78
+South Georgia and the South Sandwich Islands,77
+Greenland,73
+Mayotte,72
+Saint Helena,66
+Marshall Islands,62
+Reunion,56
+Svalbard and Jan Mayen,54
+Tonga,51
+French Southern Territories,51
+Christmas Island,48
+Samoa,46
+Norfolk Island,44
+American Samoa,43
+Macao,40
+Kiribati,39
+Cook Islands,39
+Wallis and Futuna,36
+Antarctica,34
+Pitcairn,24
+Monaco,24
+Niue,24
+Nauru,20
+San Marino,18
+Tuvalu,18
+Bouvet Island,13
+Tokelau,10
+Syria,347
+MK,320
```

Comment:
This is bugging me because the rest of the file is sorted by number of birds descending

Reply:
I'll fix that